lightningd/notification.c

```diff
@@ -175,16 +177,17 @@ REGISTER_NOTIFICATION(invoice_payment,
 		      invoice_payment_notification_serialize)
 
 void notify_invoice_payment(struct lightningd *ld, struct amount_msat amount,
-			    struct preimage preimage, const struct json_escape *label)
+			    struct preimage preimage, const struct json_escape *label, struct bitcoin_outpoint *outpoint)
 {
 	void (*serialize)(struct json_stream *,
 			  struct amount_msat,
 			  struct preimage,
-			  const struct json_escape *) = invoice_payment_notification_gen.serialize;
+			  const struct json_escape *,
+			  struct bitcoin_outpoint *) = invoice_payment_notification_gen.serialize;
```

Comment:
parameter order here differs from actual function...

Reply:
Whoops, thanks for catching this!